test/test_volatility.py

```diff
@@ -126,10 +136,7 @@ def test_windows_dumpfiles(image, volatility, python):
 
     failed_chksms = 0
 
-    if sys.platform == "win32":
-        file_name = ntpath.basename(image)
-    else:
-        file_name = os.path.basename(image)
+    file_name = os.path.basename(image)
```

Comment:
Was this unnecessary, or do we just no longer intend to run tests on windows systems?